google/gax/api_callable.py

```diff
@@ -155,6 +156,107 @@ def inner(request):
     return inner
 
 
+def _construct_bundling(method_config, method_bundling_override):
+    """Helper for ``construct_settings()``."""
+    if ('bundle_options' in method_config and
+            'bundle_descriptor' in method_config):
+
+        if method_bundling_override == OPTION_INHERIT:
+            bundler = bundling.Executor(BundleOptions(
+                **method_config['bundle_options']))
+        else:
+            if method_bundling_override:
+                bundler = bundling.Executor(method_bundling_override)
+            else:
+                bundler = None
+
+        bundle_descriptor = BundleDescriptor(
+            **method_config['bundle_descriptor'])
+
+    else:
+        bundle_descriptor = None
+        bundler = None
+    return bundler, bundle_descriptor
+
+
+def _construct_retry(
+        method_config, method_retry_override, retry_codes_def, retry_params,
```

Comment:
Although private, this method has sufficient complexity that it warrants documenting the Args and Return value.